tests/jacobi/include/helpers-jacobi.hpp

```diff
@@ -77,7 +77,7 @@ int es = PAPI_NULL; //event set
 
 size_t NB_ITER = CONST_NB_ITER;
 size_t M = WG_MULT0*J_CL_DEVICE_CONST_WORK_GROUP_SIZE0+JACOBI_DELTA;
-size_t N = WG_MULT1*J_CL_DEVICE_CONST_WORK_GROUP_SIZE1+JACOBI_DELTA;
+size_t K = WG_MULT1*J_CL_DEVICE_CONST_WORK_GROUP_SIZE1+JACOBI_DELTA;
```

Comment:
Again, the global variable named `N` conflicts with some template code which will hide the global instance of the id. (I believe buffer dimensions are also referred to with N) I used the "Rename" feature of VS here, which is a bit smarter than Find and Replace all, as it only changes the references to the instance in question, and not places where the name is hidden. (This is the perfect example when Rename is smarter than FnR.)